datanode/gateway/graphql/schema.graphql

```diff
@@ -4211,6 +4215,29 @@ type MarketConnection {
   pageInfo: PageInfo!
 }
 
+type SuccessorMarket {
+  "The market"
```

Comment:
Is this returned as an ID?

Reply:
No, the type defined on the field is `Market` so it's the full Market object from which you will be able to choose whichever fields you want returned in the gql query.